tools/benchmarks/classification/knn_imagenet/slurm_test_knn_pretrain.sh

```diff
@@ -11,7 +11,6 @@ PY_ARGS=${@:5}
 GPUS=${GPUS:-8}
 GPUS_PER_NODE=${GPUS_PER_NODE:-8}
 CPUS_PER_TASK=${CPUS_PER_TASK:-5}
-PORT=${PORT:-29500}
 SRUN_ARGS=${SRUN_ARGS:-""}
 
 # set work_dir according to config path and pretrained model to distinguish different models
```

Comment:
better to set WORK_DIR by users

Reply:
The script style follows and is consistent with the `svm_voc07` benchmark.